### DISCUSSION TITLE
posts/blog-diy

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#13] [General]
# posts/blog-diy

2014 年，我注册了这个域名用来自建博客发表文章，相继用过了 WordPress，Hexo，Hugo 等等博客系统。去年突发奇想，想要 DIY 一个。迭代更新了半年时间，收到了不少朋友们的好评。本文主要介绍一下这个博客系统开发过程中总结的经验。

http://localhost:3000/posts/blog-diy

**Comment by whexy** [comment]
把博客放在 Vercel + Azure CDN 的方案，目前一个月成本大约在 0.05USD 左右。是的你没有看错，5 美分！

**Reply by leofee** [reply]
wow，真酷。
我也想自己写个网站，可是技术有限，水平不足啊

**Reply by CodingOnStar** [reply]
请问用了CDN大概能有多少加速呢？有测试过吗

**Reply by whexy** [reply]
> 请问用了CDN大概能有多少加速呢？有测试过吗

因服务和地区差异，这类测试通常得不到准确的结论。在网络环境较好的地区，GitHub Pages/Vercel 本身服务质量就很优秀，自然也无需额外的CDN。

选用Azure CDN的主要原因是防止网络环境较差的地区不能访问GitHub Pages/Vercel的服务。应该算是一种保险手段。

**Comment by TravisRoad** [comment]
看到你的这篇博客后我大有感触，借鉴你的代码也写了一个自己的[博客](https://blog.lxythan2lxy.cn/)🤝，获益很多;D

**Reply by whexy** [reply]
Well Done！ 看了你的 Git 修改记录，能从我这一团乱码的代码（/捂脸）里设计出这么好的博客网站，实在是太厉害了！

**Comment by vicar5235** [comment]
感谢博主分享，我目前在用 WP，已经用了两年左右了，今天无意间刷到你的网站，非常的快速和简洁，但我是一个完全不懂代码的业余爱好者，请问假如我想转成博主这种风格的话，需要准备什么工作呢，可实现难度大吗？

**Comment by ganziqim** [comment]
rss地址失效了，点进去是404

**Reply by whexy** [reply]
感谢提醒！某次更新时更改了 RSS 路径，已经重新更新了文章里的地址。

**Comment by Hou-Xiaoxuan** [comment]
rss取消了吗，可惜。

**Reply by whexy** [reply]
感谢提醒！某次更新时更改了 RSS 路径，已经重新更新了文章里的地址。
